src/lib/mongoBackend/mongoGetSubscriptions.cpp

```diff
@@ -115,7 +115,7 @@ static void setSubject(Subscription* s, const BSONObj& r)
 
   if (r.hasField(CSUB_EXPR))
   {
-    mongo::BSONObj expression = getFieldF(r, CSUB_EXPR).Obj();
+    mongo::BSONObj expression = getObjectFieldF(r, CSUB_EXPR);
     std::string    q          = getStringFieldF(expression, CSUB_EXPR_Q);
```

Comment:
Surprising to see earlier changes  from `getFieldF(...).embeddedObject()` to `getObjectFieldF()` but here we go from `getFieldF().Obj()` to `getObjectFieldF()` ...   Care to explain?